### DISCUSSION TITLE
FormBuilder does not use internal mail theme

### DISCUSSION THREAD

**Discussion by betterapp** [dachcom-digital/pimcore-formbuilder#364] [Q&A] [answered]
I have a form configured like this:

![image](https://user-images.githubusercontent.com/7449124/203705856-774e8ec2-df8b-4180-b7f4-ec4f148564b4.png)

and output channel like this:
![image](https://user-images.githubusercontent.com/7449124/203706046-3cd1fea2-0b4f-4b5f-bb2c-546b6696dffa.png)

The output channel use ContactEmail document (type: email)
which is configured like this:
![image](https://user-images.githubusercontent.com/7449124/203706313-3f063b39-4c25-445b-bf43-b8233abe76a1.png)

I decide to not use file based mail template and setup Mail Layout inside FormBuilder like this:
![image](https://user-images.githubusercontent.com/7449124/203706693-08e01148-741c-445e-9c71-465d97e3180d.png)

The Form is added to document.
When I send email and click to see the message body
![image](https://user-images.githubusercontent.com/7449124/203707056-111fea79-bddc-454c-8971-1cccbb82d789.png)

I have only PimCore logo.
![image](https://user-images.githubusercontent.com/7449124/203707731-d5bf92b9-fc17-4a3b-ba83-a8ed632b00c6.png)

For me it is looks like that I need to create mail template file for ContactEmail (email type) document like always ? 
If yes do not understand what for is Mail Layout inside FormBuilder ?
How it should be configured to use mail template from FormBuilder's Mail Layout ?

**Comment by solverat** [comment, marked as answer]
Hey @betterapp. You probably don't want to use the mail editor which only generates text mails (If your recipient is an automated system for example => https://github.com/dachcom-digital/pimcore-formbuilder/blob/master/docs/OutputWorkflow/10_EmailChannel.md#mail-layout-editor)

Just use the mail template itself (in your case `/ContactEmail`). This template should use the document type "Formbuilder Email" which is available after the installation of FormBuilder.

![image](https://user-images.githubusercontent.com/700119/203722262-005df68c-fccf-48aa-9326-b2800847f75f.png)

**Reply by betterapp** [reply]
Thank. It is working.
I could not found any information in the documentation that say I should use FormBuilder Email document but not Standard PimCore Email document.
I created it before I installed this bundle and did not know that it add extra email document type to pimcore.
